cmd/prometheus/main.go

```diff
@@ -278,6 +278,11 @@ func main() {
 		os.Exit(2)
 	}
 
+	if _, err := config.LoadFile(cfg.configFile); err != nil {
+		level.Info(logger).Log("msg", fmt.Sprintf("Error loading config (--config.file=%s)", cfg.configFile), "err", err)
+		os.Exit(2)
```

Comment:
I am not really sure what the exit code should be. `TestFailedStartupExitCode` expects 1 for wrong file name. Someone aware of this can shed some light.